frontend/src/components/DataCard/CardData.jsx

```diff
@@ -29,7 +30,7 @@ function drawSkipButton(card, onSkip) {
 
     const handleSkip = (event) => {
         event.preventDefault();
-        onSkip(card, message);
+        onSkip(card, true, message);
```

Comment:
Looks like this needs to be updated in history as well. Right now when I send to administrator the reason is set as "true" or "false" instead of my reason.